axelor-crm/src/main/java/com/axelor/apps/crm/message/MessageServiceCrmImpl.java

```diff
@@ -59,38 +60,29 @@ public Message createMessage(Event event)
 
     // Get template depending on event type
     Template template = null;
+    Company company = event.getUser().getActiveCompany();
 
-    switch (event.getTypeSelect()) {
-      case EventRepository.TYPE_EVENT:
-        template =
-            Beans.get(CrmConfigService.class)
-                .getCrmConfig(event.getUser().getActiveCompany())
-                .getEventTemplate();
-        break;
+    if (company != null) {
```

Comment:
I think this method must get a company, it looks like the parent method will throw a NPE if a message was not generated. So please throw an exception if the company is not found.